frontend/public/components/factory/modal.tsx

```diff
@@ -65,11 +66,21 @@ export const createModalLauncher: CreateModalLauncher = (Component) => (props) =
 export const ModalTitle: React.SFC<ModalTitleProps> = ({
   children,
   className = 'modal-header',
+  enableCloseButton,
+  close,
 }) => (
   <div className={className}>
-    <h1 className="pf-c-title pf-m-2xl" data-test-id="modal-title">
+    <div className="pf-c-title pf-m-2xl" data-test-id="modal-title">
       {children}
-    </h1>
+      {enableCloseButton && (
```

Comment:
You can probably just use `close` prop and enable the close button if a callback is sent. No need for an extra boolean flag.

Reply:
updated to use `close`